otodom/task_1/mk/otodom_listing.json

```diff
@@ -0,0 +1,546 @@
+[
+  {
+    "url": "http://www.otodom.pl/pl/oferta/woronicza-38-5m2-lux-metro-wierzbno-stary-mokotow-ID4n5K6",
+    "otodom_id": "64608966",
+    "title": "Woronicza 38,5m2 LUX Metro Wierzbno Stary Mokotów",
+    "localization": {
+      "province": " mazowieckie",
+      "city": " Warszawa",
+      "district": " Wierzbno",
```

Comment:
you can see that sometimes the values start with addidtional space. Try to get rid of them in the code